### PR TITLE
Refactoring AWS settings dialog to UI DSL

### PR DESCRIPTION
**Pull request by manodnyab**
## Description

Switching the AWS Settings dialog from .form to ui dsl and adding a link for telemetry disclosure

 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by rli — APPROVED** [review]

File: resources/resources/software/aws/toolkits/resources/MessagesBundle.properties [hunk 1]
```diff
@@ -132,7 +132,7 @@ aws.settings.sam.show_all_gutter_icons_tooltip=When enabled, show gutter icons i
 aws.settings.serverless_label=Serverless settings
 aws.settings.show.label=Show AWS Settings
 aws.settings.telemetry.description=Help us improve our product! Allow us to collect basic usage data.
-aws.settings.telemetry.option=Send usage metrics to AWS
+aws.settings.telemetry.option=<html>Send usage metrics to AWS <a href=" https://docs.aws.amazon.com/sdkref/latest/guide/support-maint-idetoolkits.html">(details)</a></html>
```
[the remark this answers is not in the document]
Comment (manodnyab) [reply resources/resources/software/aws/toolkits/resources/MessagesBundle.properties]: Done: https://github.com/aws/aws-toolkit-jetbrains/pull/3848/files#diff-e33d0539be12806a9448a56c2e93ae2e86f80c5b5c99dbbb64b6b3cf0037e3f3R72

File: jetbrains-core/src/software/aws/toolkits/jetbrains/settings/AwsSettingsConfigurable.kt [hunk 2]
```diff
@@ -30,58 +28,64 @@ import java.nio.file.Path
...
+                }
 
-    init {
-        applicationLevelSettings.border = IdeBorderFactory.createTitledBorder(message("aws.settings.global_label"))
-        serverlessSettings.border = IdeBorderFactory.createTitledBorder(message("aws.settings.serverless_label"))
-        SwingHelper.setPreferredWidth(samExecutablePath, panel.width)
+                val enableTelemetryText = "<html>${message("aws.settings.telemetry.option")} <a href=\"https://docs.aws.amazon.com/sdkref/latest/guide/support-maint-idetoolkits.html\">${message(
+                    "general.details"
+                )}</a></html>"
+                cell(JBLabel(enableTelemetryText).setCopyable(true))
```
Comment (rli) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/settings/AwsSettingsConfigurable.kt:70]: browserLink(message(...), https://...)

Comment (rli) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/settings/AwsSettingsConfigurable.kt:70]: or
```
text("Send usage metrics to AWS <a>(details)</a>") { BrowserUtil.open(...) }
```

**Review by rli — APPROVED** [review]

**Pull request merged**